docs/ref/contrib/admin/index.txt

```diff
@@ -2861,6 +2861,43 @@ Templates can override or extend base admin templates as described in
 ``AdminSite`` methods
 ---------------------
 
+.. method:: AdminSite.get_app_list(request, app_label=None)
```

Comment:
`AdminSite` methods are alphabetized, please move it below `each_context()` docs.